<launch>
  <node pkg="mp_tracker" type="d_tracker_plus_kinect_node" name="phd_kinect_tracker" output="screen"><!-- launch-prefix="gdb -ex run - -args" -->
    <param name="tracking_frame" value="/odom"/>
    <param name="laser_topic" value="/scan"/>
    <param name="pointcloud_topic" value="/openni/depth/points"/>
    <param name="camera_topic" value="/openni/rgb/image_color"/>
    <param name="camera_info_topic" value="/openni/rgb/camera_info"/>
    <param name="use_beta_mixture" value="false"/>
    <param name="use_optimal_beta_merge" value="false"/>
    <param name="merge_beta_threshold" value="0.04"/>
    <param name="merge_gauss_threshold" value="0.1"/>
    <param name="birth_prior_weight" value="0.01"/>
    <param name="clutter" value="0.25"/>
    <param name="detection_certainty" value="0.4"/>
    <param name="max_components" value="200"/>
    <param name="debug_script" value=""/>
  </node>
</launch>
